public/data/devesh2511.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Devesh Jain",
+  "type": "personal",
+  "bio": "I build things that can't be touched, but can be felt, can be used, can be helpful.. Committing codes and mistakes..",
+  "avatar": "https://github.com/devesh2511.png",
+  "links": [
+    
```

Comment:
Please remove this extra line (no need to delete pull request - if you change the code, commit it and push it to your branch the PR will update automatically)